CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.4.0
+-----
+### Enhancements
+* Maintainability
+    * Added new `mftf static-checks` command to run existing static checks against the attached test codebase
+        * See DevDocs for details
+    * Added new `mftf generate:docs` command that generates documentation about attached test codebase
+        * See DevDocs for details
+* Traceability
+    * Allure reports for tests now contain collapsible sections for `actionGroup`s used in execution.
+
+### Fixes
+* Fixed an issue where `magentoCli` would treat `argument="0"` as a null value.
+* Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly.
```

Comment:
Not sure if you need to be explicit, the `amOnPage` is also PWA specific.

Reply:
I didn't work on the ticket so I was summarizing best as I could understand. How would you word this line instead?

Reply:
Something like:
Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly when MagentoPwaWebDriver is enabled.

Reply:
Sounds good 👍